test/Template.spec.js

```diff
@@ -46,4 +46,25 @@ describe('Template', function() {
 
   });
 
+  describe('test()', function() {
+
+    it('should test test endpoint', function() {
+      expect( t ).to.respondTo('test');
+      return expect( t.test(testData) ).to.eventually.equal('Generation successful');
+    });
+
+    it('should test content', function() {
+      t.test({}).catch(function(err) {
```

Comment:
A more canonical way to do this is probably to use the `.to.eventually.throw()` [syntax](http://chaijs.com/api/bdd/#method_throw)